packages/sn-repository-events/src/EventHub.ts

```diff
@@ -120,6 +120,7 @@ export class EventHub implements Disposable {
    * Triggered after copying a content has been failed
    */
   public readonly onContentCopyFailed = new ObservableValue<ContentCopyFailed>()
+  public readonly onUpload = new ObservableValue<{ isUploaded: boolean }>()
```

Comment:
```suggestion
  public readonly onUploadFinished = new ObservableValue<{ isUploaded: boolean }>()
```
I think an onUploadFailed can be also useful.

Reply:
👍